Specs/Number/Japanese/NumberModel.json

```diff
@@ -3153,7 +3148,7 @@
     "Input": "答えは−３５分の１００です。",
     "IgnoreResolution": true,
     "NotSupportedByDesign": "javascript, java, python",
-    "NotSupported": "dotnet",
+    
```

Comment:
Nitpick: Please remove blank lines from the specs.

Reply:
Sure @tellarin . That will be fixed in coming PR.

Reply:
Fixed in https://github.com/microsoft/Recognizers-Text/pull/2109